tiledb/datatypes.py

```diff
@@ -24,6 +24,13 @@ def from_numpy(cls, dtype: np.dtype) -> DataType:
         if dtype == "blob":
             return cls(np.dtype("S"), lt.DataType.BLOB, 1)
 
+        if hasattr(lt.DataType, "GEOM_WKB"):
+            if dtype == "wkb":
+                return cls(np.dtype("S"), lt.DataType.GEOM_WKB, 1)
+
+            if dtype == "wkt":
+                return cls(np.dtype("S"), lt.DataType.GEOM_WKT, 1)
+
```

Comment:
Should we have a separate branch for WKT? 

```suggestion
        if hasattr(lt.DataType, "GEOM_WKB"):
            if dtype == "wkb":
                return cls(np.dtype("S"), lt.DataType.GEOM_WKB, 1)

        if hasattr(lt.DataType, "GEOM_WKT"):
            if dtype == "wkt":
                return cls(np.dtype("S"), lt.DataType.GEOM_WKT, 1)

```